homeassistant/components/opengarage/cover.py

```diff
@@ -49,11 +49,12 @@
     }
 )
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+PLATFORM_SCHEMA = cv.deprecated(
```

Comment:
This is incorrect. It should be `vol.All(cv.deprecated(DOMAIN), PLATFORM_SCHEMA.extend(…))`

Reply:
actually, cv.deprecated doesn't work for PLATFORM_SCHEMA at all. It only works with top level schemas. Just print a warning in async_setup_platform.